Sources/MapboxCoreNavigation/TileStore.swift

```diff
@@ -0,0 +1,95 @@
+import MapboxNavigationNative
+
+
+extension TileStore {
+    public typealias ContainsCompletion = (Bool?) -> Void
```

Comment:
Maybe it's better to use `Result<Bool, Error>` here?

Reply:
That would introduce some inconsistency. All `TileStore` methods swift wrappers are defined in `MapboxMaps` module and have `Result` argument. They also introduce their defined `Error` types. Thus, if I am going to also use `Error`s, that would be another type (since I cannot import maps here for reuse) which may be implicit and confusing for users. So I decided to make it explicitly different by using another completion type.